### ISSUE TITLE
L2: handle `unwraps` in prover_server.rs

### ISSUE THREAD

**Issue by fborello-lambda** [lambdaclass/lambda_ethereum_rust#1168]
Handle unwraps in `prover_server.rs` we may want to handle unwraps in `prover_client.rs` and even globally.

**Comment by varun-doshi** [comment]
Is this still open?

**Comment by fborello-lambda** [comment]
> Is this still open?

Yes, but we are working on this. It is becoming more complex than just handling the unwraps.

Thanks for your interest in helping!
